py-scripts/lf_ftp.py

```diff
@@ -2096,7 +2484,58 @@ def pass_fail_duration(band, file_size):
             duration = float(duration)
 
         return duration
-
+    # Get group and profile values from arguments and convert comma-separated strings into lists
+    if (args.group_name is not None):
+        selected_groups = args.group_name.split(',')
+    else:
+        selected_groups = []  # Default to empty list if group name is not provided
+    if (args.profile_name is not None):
+        selected_profiles = args.profile_name.split(',')
+    else:
+        selected_profiles = []  # Default to empty list if profile name is not provided
+
+    if args.clients_type == 'Real' and args.config and args.group_name is None:
```

Comment:
Please move argument validation to a helper function. Keeping everything in `main()` leads to poor readability over time.